subprojects/vinumc/tests/library_test.c

```diff
@@ -0,0 +1,33 @@
+#include <vunit.h>
+
+void test_call(struct vunit_test_ctx *ctx) {
+	char *out = NULL;
+
+	vunit_run_vinumc_ok(ctx,
+		"[day_of_week 2024-09-26]\n",
+		&out,
+		"--with","subprojects/stdlib/libstdlib.so" , NULL
```

Comment:
hum, maybe create an external lib just for testing inisde the vinumc project. If you want to test the stdlib behavior (which we probably should) create a test dir on the stdlib project.

Reply:
that was the purpose of the stdlib of this pr

Reply:
I see, but I don't like that vinumc tests depend on another project.

Create another lib inside `test` for isolating the two projects. For now, it could be just a dumb function that returns some string, for the current tests it will suffice. 

The stdlib part of this PR will be the starting point.